G4ShareSpy/G4CGMManagerSettingsNavigationController.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  G4CGMManagerSettingsNavigationController.swift
+//  G4ShareSpy
+//
+//  Created by Nathaniel Hamming on 2021-01-15.
+//  Copyright © 2021 Mark Wilson. All rights reserved.
+//
+
+import UIKit
+import HealthKit
+import LoopKit
+import LoopKitUI
+
+class G4CGMManagerSettingsNavigationController: SettingsNavigationViewController, PreferredGlucoseUnitObserver {
```

Comment:
Recommend ditching this and using generic `CGMManagerSettingsNavigationController` described in my `LoopKit` PR comments.